index.coffee

```diff
@@ -32,7 +33,11 @@ module.exports = (grunt) ->
 
     {certificateFile, certificatePassword} = config
 
-    appMetadata = grunt.file.readJSON(path.join(appDirectory, 'resources', 'app', 'package.json'))
+    appDir = path.join(appDirectory, 'resources', 'app')
```

Comment:
How about calling this `appResourcesDirectory`? Having `appDir` and `appDirectory` is kind of confusing.